the-bureau/bureau-structure/index.html

```diff
@@ -12,7 +12,13 @@
 
 {% block css -%}
     {{ super() }}
-    <link rel="stylesheet" href="/static/vendor/slick-carousel/slick.css">
+    {# TODO: Resolve slick font issue.
+             For the slick stylesheet to pick up its font and img reference the
+             correct URL should be /src/vendor/slick-carousel/slick/slick.css
+             However, including the slick fonts makes the dots on the
+             /the-bureau/ page at mobile sizes enormous.
+    #}
+    <link rel="stylesheet" href="/src/vendor/slick-carousel/slick.css">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/e34401e4933f418bb4ee686b4e695c750f6e3b9b